src/searching.jl

```diff
@@ -126,3 +126,15 @@ function search(searcher::Searcher, query::String, k::Int)
     pids, scores = pids[indices], scores[indices]
     pids[1:k], scores[1:k]
 end
+
```

Comment:
Similar to the `Indexer`: we can get rid of any configuration parameters (like the dimension, nprobe, ncandidates etc). I like that it shows that total number of embeddings and centroids!